osu.Game/Overlays/Chat/DrawableUsername.cs

```diff
@@ -65,6 +67,9 @@ public override float Width
         [Resolved(canBeNull: true)]
         private UserProfileOverlay? profileOverlay { get; set; }
 
+        [Resolved]
+        private Bindable<Channel>? currentChannel { get; set; }
```

Comment:
This should *really* be
```suggestion
        private Bindable<Channel?>? currentChannel { get; set; }
```

Both `ChatOverlay`'s and `StandaloneChatDisplay`'s cached bindables can contain nulls.

https://github.com/ppy/osu/blob/53d968499eba134545ec8d6e5dc4cd84346b4f21/osu.Game/Overlays/ChatOverlay.cs#L66-L67
https://github.com/ppy/osu/blob/53d968499eba134545ec8d6e5dc4cd84346b4f21/osu.Game/Online/Chat/StandAloneChatDisplay.cs#L117

As to whether a `DrawableUsername` will *ever* actually see a bindable with null inside - I don't know, but I'd rather play it safe.